### DISCUSSION TITLE
Is the mail storage encrypted yet?

### DISCUSSION THREAD

**Discussion by MartinTheDragon** [emersion/maddy#611] [Q&A]
I couldn't find any information about it other than #75 and #10, and the code is a bit hard to follow (I'm not exactly familiar with Go either).
The first issue is closed in favor of the second, and the second one is still open and has been stale since 2020.
I want the emails to be protected from the server host itself, so that's kinda important.

**Comment by foxcpp** [comment]
No, it is currently not implemented.
